tests/integration_tests_control_files/euler_integration/naca0012/CMakeLists.txt

```diff
@@ -12,6 +12,13 @@ add_test(
   WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 )
 
+configure_file(naca0012.prm naca0012.prm COPYONLY)
+add_test(
+        NAME NACA0012_FLOWSOLVER
```

Comment:
Maybe give this a better name that's a bit more descriptive, like `INVISCID_TRANSONIC_STEADY_STATE_NACA0012`

Reply:
Fixed in most recent commit.